helix-rest/src/main/java/org/apache/helix/rest/server/HelixRestServer.java

```diff
@@ -94,8 +101,21 @@ public HelixRestServer(List<HelixRestNamespace> namespaces, int port, String url
     init(namespaces, port, urlPrefix, auditLoggers);
   }
 
+  public HelixRestServer(List<HelixRestNamespace> namespaces, int port, String urlPrefix,
+      List<AuditLogger> auditLoggers, AuthValidator clusterAuthValidator,
+      AuthValidator namespaceAuthValidator) {
+    init(namespaces, port, urlPrefix, auditLoggers, clusterAuthValidator, namespaceAuthValidator);
+  }
+
   private void init(List<HelixRestNamespace> namespaces, int port, String urlPrefix,
       List<AuditLogger> auditLoggers) {
+    init(namespaces, port, urlPrefix, auditLoggers, new DefaultAuthValidator(),
+        new DefaultAuthValidator());
```

Comment:
Curious, does it require different instances?

Reply:
Behavior-wise, not really; meaning-wise, they should be separate (1 for cluster, 1 for namespace). I can change it if you insist. :)

Reply:
Not necessary, I don't have strong opinion, let's keep it as is.
I was thinking of enforcing singleton, but realized it's a more implementation specific thing, and probably should leave it to users.